src/app/table/table.component.css

```diff
@@ -0,0 +1,95 @@
+table {
+  border: 1px solid #ccc;
+  border-collapse: collapse;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  table-layout: fixed;
+}
+
+table caption {
+  font-size: 1.5em;
+  margin: .5em 0 .75em;
+}
+
+table tr {
+  background-color: #f8f8f8;
+  border: 1px solid #ddd;
+  cursor: pointer;
+  padding: .35em;
+}
+table tr.bgColor {
+  background-color: lightgreen;
+}
+
+table th,
+table td {
+  padding: .625em;
+  text-align: center;
+}
+
+table th {
+  font-size: .85em;
+  letter-spacing: .1em;
+  text-transform: uppercase;
+}
+
+@media screen and (max-width: 600px) {
```

Comment:
style lepiej dostarczać za pomocą metody mobile-first ale nie jest to konieczne na naszych zajęciach :)